### ISSUE TITLE
Can I use it offline?

### ISSUE THREAD

**Issue by qfb594** [ashawkey/stable-dreamfusion#102]
Can I use it offline? I don't want to be connected to the Internet and want to use the sd version of the ai painting instead of the token,

**Comment by ashawkey** [comment]
@qfb594 Yes, once you have downloaded the checkpoint from hugging face, it could be used offline.
